packages/web-forms/src/components/controls/RankControl.vue

```diff
@@ -205,6 +231,10 @@ const swapItems = (index: number, newPosition: number) => {
 		align-items: center;
 		gap: var(--rankSpacing);
 	}
+
+	.rank-label svg {
+		flex-shrink: 0;
```

Comment:
Prevents icons from shrinking when text is long and needs to wrap to the next line